gapic-common/lib/gapic/grpc/status_details.rb

```diff
@@ -13,23 +13,16 @@
 # limitations under the License.
 
 require "grpc"
+require "grpc/errors"
 require "grpc/google_rpc_status_utils"
-require "gapic/errors"
 require "google/protobuf/well_known_types"
 # Required in order to deserialize common error detail proto types
 require "google/rpc/error_details_pb"
 
-module Gapic
-  class GapicError < StandardError
+module GRPC
+  class BadStatus < StandardError
```

Comment:
I suppose this could be considered out of scope for this PR, but we should add docs to this method. Can we at least `TODO` for a doc pass later?